src/main/java/com/catapi/controller/BreedController.java

```diff
@@ -0,0 +1,23 @@
+package com.catapi.controller;
+
+import com.catapi.service.BreedService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/breed")
+public class BreedController {
+    private final BreedService breedService;
+
+    public BreedController(BreedService breedService) {
+        this.breedService = breedService;
+    }
+
+    @GetMapping("/update")
+    public ResponseEntity<String> updateBreedDbController() {
+        breedService.updateBreedDb();
```

Comment:
I don't like the method name "updateBreedDb" and "updateBreedDbController"

It discribes nothing. It's much better to name it "updateBreedsFromExternalApi" in both cases (service and controller);

Reply:
Done